### ISSUE TITLE
azure-search-documents beta.4 has unpublished test dependency

### ISSUE THREAD

**Issue by scbedd** [Azure/azure-sdk-for-java#13097]
Onboarded beta.4. Results:

```
Failed to execute goal on project azure-search-documents: Could not resolve dependencies for project 
com.azure:azure-search-documents:jar:1.0.0-beta.4: Could not find artifact com.azure:azure-test-watcher:jar:1.0.0-beta.1 
in ossrh(https://oss.sonatype.org/content/repositories/snapshots/) -> [Help 1]
```

Need to execute same methodology to put the test deps into a specific profile that isn't pulled in by default.

**Comment by alzimmermsft** [comment]
This dependency was removed in https://github.com/Azure/azure-sdk-for-java/pull/12989

**Comment by scbedd** [comment]
@alzimmermsft if the 11.0.0 package that just went out doesn't have this issue, we should just close this.

**Comment by scbedd** [comment]
Resolved in azure-search-documents version 11.0,0. Closing.
